src/modules/Search/Search.js

```diff
@@ -508,11 +516,12 @@ export default class Search extends Component {
   // ----------------------------------------
 
   renderSearchInput = () => {
-    const { icon, placeholder } = this.props
+    const { icon, placeholder, input } = this.props
     const { value } = this.state
 
     return (
       <Input
+        fluid={input.fluid}
```

Comment:
@levithomason - Given the code in this snippet, if you pass any `input` prop it will override the default `input` props. For example:

```jsx
<Search>
// yields an input of
<input type="text" value="" class="prompt" tabindex="0" autocomplete="off">

<Search input={{fluid: true}} />
// would yield an input of
<input type="text" value="" class="fluid">
// but we'd desire
<input type="text" value="" class="prompt fluid" tabindex="0" autocomplete="off">
```